scss/base/_colors.scss

```diff
@@ -135,6 +150,31 @@ $purple-dark: #582e63;
 .bg-purple-dark { background-color: $purple-dark; }
 .bc-purple-dark { border-color: $purple-dark; }
 
+$ds-grey-10:  #f9f9f9;
+.ds-grey-10 { color: $ds-grey-10; }
+.bg-ds-grey-10 { background-color: $ds-grey-10; }
+.bc-ds-grey-10 { border-color: $ds-grey-10; }
+
+$ds-grey-20:  #f3f3f4;
+.ds-grey-20 { color: $ds-grey-20; }
+.bg-ds-grey-20 { background-color: $ds-grey-20; }
+.bc-ds-grey-20 { border-color: $ds-grey-20; }
+
```

Comment:
What is the migration plan once this is merged? I assume swapping all existing the classes with the DS ones?

I wonder if it would be easier to keep the existing classes but reassign the colors. Then we could replace the "deprecated" classes with the new ones. So for example:

```css
.blue-70 { color: $ds-blue-40 }
.blue-50 { color: $ds-blue-40 }
.blue-40 { color: $ds-blue-40 }
```

When that's merged we replace all usages of `.blue-50` and `.blue-70` in the monolith with `.blue-40`.

Reply:
yes it would make life easier but it would be inconsistent with the colors in the monolith bc those too are being updated. Someone may see `className="bold blue-70"` and assume it's the blue-70 in `colors.scss`. However your suggestion _would_ be easier and safer. 

What do you think?

Reply:
> Someone may see className="bold blue-70" and assume it's the blue-70 in colors.scss

If this is the main concern I'd say it's worth the risk. It's mostly us using the colors anyway and we can communicate that things are mis-matched at the moment.